src/pages/memberhsip/components/youthProfileForm/YouthProfileForm.tsx

```diff
@@ -54,6 +55,25 @@ const SignupSchema = Yup.object().shape({
 function RegistrationForm(props: Props) {
   const { t } = useTranslation();
   const languages = ['Suomi', 'Svenska', 'English', 'Other'];
+  const photoPermit = ['Kyllä', 'Ei'];
+
+  const getYearDiff = (year: string, month: string, day: string) => {
+    if (
+      year !== null &&
```

Comment:
`null` and empty string are falsy values in javascript, this whole check could be shortened to
```
if (year && Number(year) >= 1900 && month && day) {
  ...
}
```